dev-client/src/translations/en.json

```diff
@@ -424,21 +426,39 @@
       "photos": "Photos",
       "notes": "Notes"
     },
+    "data": {
+      "rockFragmentVolume": "Rock fragment",
+      "texture": "Texture",
+      "colorValue": "Color"
+    },
+    "soil_preset": {
+      "header": "Change soil pit depths",
+      "info": "Data already entered for a depth will be deleted when the depth interval is changed.",
+      "label": "Depth Intervals",
+      "LANDPKS": "LandPKS (default)",
+      "NRCS": "NRCS",
+      "CUSTOM": "Custom"
```

Comment:
```suggestion
      "landpks": "LandPKS (default)",
      "nrcs": "NRCS",
      "custom": "Custom"
```

Reply:
there are many places in the translation file (including here) where keys are uppercase because they correspond to enum values in the codebase, and it's a good indication that that key in the translation file might be referenced programmatically, i don't think we need to have them all lowercase